src/info/user.go.tmpl

```diff
@@ -21,31 +21,47 @@
 	{{ $user = $member.User }}
 {{ end }}
 
-{{ $roleMentions := cslice }}
-{{ range $member.Roles }} {{- $roleMentions = $roleMentions.Append (print "<@&" . ">") -}} {{ end }}
+{{ $roles := $member.Roles }}
+{{ $rolesDict := dict }} {{ $rolesPos := cslice }}
+{{ range $roles }}
+	{{- $role := $.Guild.GetRole . }}
+	{{- $rolesDict.Set $role.Position $role.ID }}
+	{{- $rolesPos = $rolesPos.Append $role.Position -}}
+{{ end }}
```

Comment:
You can sort a slice of `sdict "role" [role object] "position" [position]`, providing `sort` with the option `key=position` instead of building `$rolesPos` and then recovering the roles via `$rolesDict`.

Reply:
I see, very good then. I read the docs on `sort` to see the best way about it but did not understand fully what it meant. Will change soon!